src/nomad_material_processing/general/schema.py

```diff
@@ -0,0 +1,278 @@
+    """
+
+    pass
+
+
+class EtchingStep(ActivityStep):
+    """
+    A step of etching process.
+    """
+
+    m_def = Section()
+    duration = Quantity(
+        type=np.float64,
+        description='The elapsed time since the annealing process started.',
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.NumberEditQuantity, defaultDisplayUnit='minute'
+        ),
+        unit='second',
+    )
+    temperature = Quantity(
+        type=np.float64,
+        description='The temperature of the etching process.',
+        a_eln={'component': 'NumberEditQuantity', 'defaultDisplayUnit': 'celsius'},
+        unit='celsius',
+    )
+    agitation = Quantity(
+        type=MEnum(
+            'Magnetic Stirring',
+            'Sonication',
+        ),
+        description='The agitation method used during the etching process.',
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.EnumEditQuantity,
+        ),
+    )
+    etching_reagents = SubSection(section_def=CompositeSystem, repeats=True)
+
+
+class Etching(Process, EntryData):
+    """
+    Class autogenerated from yaml schema.
+    """
+
+    m_def = Section(
+        a_eln=None,
+    )
+    recipe = Quantity(
+        type=Recipe,
+        description=""" The recipe used for the process. If a recipe is found,
+           all the data is copied from the Recipe within the Process.
+           """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.ReferenceEditQuantity,
+        ),
+    )
+    steps = SubSection(
+        description="""
+        The steps of the etching process.
+        """,
+        section_def=EtchingStep,
+        repeats=True,
+    )
+
+
+class EtchingRecipe(Etching, Recipe, EntryData):
+    """
+    A Recipe for an etching process.
+    """
+
+    m_def = Section(
+        a_eln={'hide': ['datetime', 'samples']},
+    )
+
+
+class Annealing(Process, EntryData):
+    """
+    Class autogenerated from yaml schema.
+    """
+
+    m_def = Section(
+        a_eln={'hide': ['steps']},
+    )
+    recipe = Quantity(
+        type=Recipe,
+        description=""" The recipe used for the process. If a recipe is found,
+           all the data is copied from the Recipe within the Process.
+           """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.ReferenceEditQuantity,
+        ),
+    )
+    temperature = Quantity(
+        type=np.float64,
+        description='The temperature of the annealing process.',
+        a_eln={'component': 'NumberEditQuantity', 'defaultDisplayUnit': 'celsius'},
+        unit='celsius',
+    )
+    duration = Quantity(
+        type=np.float64,
+        description='The elapsed time since the annealing process started.',
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.NumberEditQuantity, defaultDisplayUnit='minute'
+        ),
+        unit='second',
+    )
+    annealing_reagents = SubSection(
```

Comment:
What are annealing reagents?

Reply:
I don't know.. but someone had that. I would leave them

Reply:
But what? You are heating something, what are the reagents here? I think it is very confusing to keep them there without an explanation of what it is.